### PR TITLE
Issue structure

### PR DESCRIPTION
**Pull request by nebrius**
A proposal for what we want issues to look like for this repo and an effort to help keep discussions on topic.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+## Filing Issues
+
+Issues filed for this repository must meet the following requirements and structure.
+
+### Requirements
+
+The Inclusivity Working Group exists to increase inclusivity and diversity within the Node.js collaborator base. These discussions often touch on a wide range of topics which makes it easy for discussions to get off topic. The following requirements exist for filing issues to help make sure discussions stay on topic.
+
+- Issues should be filed with the intention of actionable steps being taken to further the mission of this working group
+    - Issues in this repository are not the place for, e.g., individual education about inclusivity/diversity or discussing whether or not this group should exist.
```
Comment (scottgonzalez) [inline CONTRIBUTING.md]: The use of e.g. doesn't make sense here, it should just be removed.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+- Issues should be filed for topics that are within the scope of this Working Group
+    - For example, issues should only be filed that request changes at the TSC level if it is an inclusivity/diversity related topic _and_ members of this working group are needed to spearhead the initiative. In all other cases, the issue should be filed under the [TSC repository](https://github.com/nodejs/tsc).
```
Comment (scottgonzalez) [inline CONTRIBUTING.md]: For someone unfamiliar with the working group and committee structure, this first reference to TSC might be confusing. I'd suggest spelling out the committee name.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+### Structure
+
+Each issue should have the following information.
+
+#### Purpose
+
+The purpose should be a brief description of what you would like to see addressed. The type of issue should be explicitly mentioned in the purpose:
+
+- _Question_ Some of the policies and recommendations of this group may not be clear, and a question asks for clarification. The resolution to a question should be an answer in the issue, not a change to policy or documentation. This issue may lead to the filing of a new request for changes issue.
+- _Request for changes to a policy_ A request for change is used to clarify an existing policy, e.g. "This document is unclear in the case of x. Can we modify it to be more explicit?" or "The document says x, but I think it would be more effective if we did y instead".
```
Comment (scottgonzalez) [inline CONTRIBUTING.md]: Missing comma after "e.g."

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,5 +1,43 @@
...
+- _Request for new policy_ This WG's equivalent of a feature request, a request to create new policies.
+
+#### Proposal
+
+This optional section is for request for changes or new policies.
```
Comment (scottgonzalez) [inline CONTRIBUTING.md]: This wording is odd. Perhaps "This optional section only applies to policy issues."

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,5 +1,43 @@
...
+
+If you have an idea of what the policy should look like, include a summary of it here. This does not need to be a fully formed version of the policy, just an overview of what you think it should look like. If you have a more fully formed proposal, please submit it as a pull request.
+
+#### Focus
+
+This section defines what is "on topic" and what is "off topic." The discussions in this group touch on a broad range of topics, and it's very easy to get off topic no matter how well intentioned everyone is. By having an explicit definition of what is considered on topic and off topic, and what the issue auther hopes to get out of the discussion, we should be able to focus better.
```
Comment (scottgonzalez) [inline CONTRIBUTING.md]: s/auther/author/

File: CONTRIBUTING.md (hunk 1 above)
Comment (Trott) [inline CONTRIBUTING.md]: If it should be kept, it should be replaced with *for example* rather than *e.g.*.

File: CONTRIBUTING.md (hunk 3 above)
Comment (Trott) [inline CONTRIBUTING.md]: *e.g.* is basically never as clear as simply spelling out *for example* so I'd suggest that all instances of it be spelled out.

**Comment by nebrius** [comment]
Thanks for the feedback, good points all around. I reworded some of the sentences that used `e.g.` to either make `for example` flow better, or removed it all together. Let me know what you think!

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,5 +1,43 @@
...
+
+### Structure
+
+Each issue should have the following information.
+
+#### Purpose
+
+The purpose should be a brief description of what you would like to see addressed. The type of issue should be explicitly mentioned in the purpose:
+
+- _Question_ Some of the policies and recommendations of this group may not be clear, and a question asks for clarification. The resolution to a question should be an answer in the issue, not a change to policy or documentation. This issue may lead to the filing of a new request for changes issue.
```
Comment (Fishrock123) [inline CONTRIBUTING.md]: Are we sure another *issue* is necessary if something ends up needing to clarified in the documentation?

Having a second issue required seems like it may be counterproductive. We can always rename the issue to clarify what it actually asks.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,5 +1,43 @@
-Even though we have a charter, this WG is essentially still in its early
-stages.
-If you are interested in helping shape this WG, have a look at the issues and PRs.
+Even though we have a charter, this WG is essentially still in its early stages. If you are interested in helping shape this WG, have a look at the issues and PRs.
```
Comment (Fishrock123) [inline CONTRIBUTING.md]: "please have"?

**Comment by Fishrock123** [comment]
The bottom of the document sounds like a template but it's not that clear about that.

Could we add an example at the bottom within code blocks to show the markdown (and be copyable)?

File: CONTRIBUTING.md (hunk 6 above)
Comment (nebrius) [inline CONTRIBUTING.md]: Good call. I'll reword to say that issue authors can file a new issue, or rename the original, at their discretion

**Comment by nebrius** [comment]
I added some examples, ptal. I also rebased against master to fix merge conflicts.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,9 +1,110 @@
-Even though we have a charter, this WG is essentially still in its early
-stages.
-If you are interested in helping shape this WG, have a look at the issues and PRs.
+Even though we have a charter, this WG is essentially still in its early stages. If you are interested in helping shape this WG, please have a look at the issues and PRs.
 
 _TODO: Add starting points_
```
Comment (Charlotteis) [inline CONTRIBUTING.md]: What's a starting point in this context? Do you mean a list of linkable contents?

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,9 +1,110 @@
...
+Issues filed for this repository must meet the following requirements and structure.
+
+### Requirements
+
+The Inclusivity Working Group exists to increase inclusivity and diversity within the Node.js collaborator base. These discussions often touch on a wide range of topics which makes it easy for discussions to get off topic. The following requirements exist for filing issues to help make sure discussions stay on topic.
+
+- Issues should be filed with the intention of actionable steps being taken to further the mission of this working group
+    - Issues in this repository are not the place for individual education about inclusivity/diversity, discussing whether or not this group should exist, etc.
+- Issues should be filed for topics that are within the scope of this Working Group
+    - For example, if you want to file an issue with project-wide scope, it should probably be filed under the [Technical Stearing Comittee repository](https://github.com/nodejs/tsc). Issues should only be filed that request changes at the TSC level here if it is an inclusivity/diversity related topic _and_ members of this working group are needed to spearhead the initiative.
```
Comment (Charlotteis) [inline CONTRIBUTING.md]: change 'stearing' to 'steering'

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -1,9 +1,110 @@
...
+- Issues should not be filed for topics that have already been decided
+    - Not everyone will agree with everything that has been decided by this group, this is expected.
+    - Issues that are filed in an attempt to reopen a discussion that has already been decided will be closed without discussion.
+    - Other moderation actions may be taken if the issue filed violates the Code of Conduct.
```
Comment (Charlotteis) [inline CONTRIBUTING.md]: Link to the Code of Conduct here maybe?

**Comment by Charlotteis** [comment]
Looks great to me. Having a contents section to get to the bits of the doc you want would be great, but that might be what is meant by 'starting point' and will likely be completed once the doc itself has been okay'ed. 

Thanks @nebrius, this is really cool.

File: CONTRIBUTING.md (hunk 10 above)
Comment (ashleygwilliams) [inline CONTRIBUTING.md]: good idea

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -1,9 +1,110 @@
...
+
+#### Focus
+
+This section defines what is "on topic" and what is "off topic." The discussions in this group touch on a broad range of topics, and it's very easy to get off topic no matter how well intentioned everyone is. By having an explicit definition of what is considered on topic and off topic, and what the issue author hopes to get out of the discussion, we should be able to focus better.
+
+### Examples
+
+Here is an example for a question issue:
+
+```
```
Comment (ashleygwilliams) [inline CONTRIBUTING.md]: code blocks don't respect soft wrap so this is a bit difficult to read, consider hard wrapping at 80cols, or probably easier/better, just use `>` and quote it instead of using a code block.

this is an amazing example. are you familiar with the grilled cheese copy pasta? https://www.reddit.com/r/copypasta/comments/2os2lp/its_about_ethics_in_grilled_cheese_sandwich_making/

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -1,9 +1,110 @@
...
+Each issue should have the following information.
+
+#### Purpose
+
+The purpose should be a brief description of what you would like to see addressed. The type of issue should be explicitly mentioned in the purpose:
+
+- _Question_ Some of the policies and recommendations of this group may not be clear, and a question asks for clarification. The resolution to a question should be an answer in the issue, not a change to policy or documentation.
+- _Request for changes to a policy_ A request for change is used to clarify an existing policy. Issues of this type are typically of the form "This document is unclear in the case of x. Can we modify it to be more explicit?" or "The document says x, but I think it would be more effective if we did y instead".
+- _Request for new policy_ This WG's equivalent of a feature request, a request to create new policies.
+
```
Comment (ashleygwilliams) [inline CONTRIBUTING.md]: we make more than policies! i would also like to see things like "let's start a new program"

**Comment by ashleygwilliams** [comment]
this is awesome and a great start. i wonder if this should be a document on it's own called "policy issues" linked to from the contributing..

i'm a fan of small documents that are all grouped together with a ToC. the primary info in contributing should be where to start so maybe an explanation of who we are, what to read, and then links to how to contribute policy changes, and also how to submit an idea for a new program.

thoughts?

**Comment by ashleygwilliams** [comment]
i'd like to get this merge asap, so if this is put into a different file(`doc/POLICY_ISSUES.md`)  or something and then linked in the top level `CONTRIBUTING.md` i can work on a `doc/PROGRAM_ISSUES`. and we'd be good to go, thoughts?

**Comment by ashleygwilliams** [comment]
oh also. i would really like a line, in bold, at the top of contributing that says "PEOPLE OVER PEDANTRY" which means if you are overwhelmed by these rules just contribute and we'll help you find the right way.

**Comment by Charlotteis** [comment]
> i'm a fan of small documents that are all grouped together with a ToC. the primary info in contributing should be where to start

agree, a long document is less likely to be read (unless you are me who has spent 6 hours in totaly so far reading this inclusivity stuff :P)

**Comment by Charlotteis** [comment]
Oh! I just recalled a great thing you can do help issue creation that we can add to this.

Over in the Hoodie Editorial team, if someone wants to pitch a blog post they can click [this link](https://github.com/hoodiehq/editorial/issues/new?title=Post%20Proposal:%20Your%20Title%20Here&body=https://github.com/hoodiehq/editorial/issues/new?title=Post%20Proposal:%20Your%20Title%20Here&body=%60%60%60md%0A%3E%20What%20is%20your%20blog%20post%20about%3F%20Describe%20it%20in%20about%20one%20paragraph.%0A%0A*Replace%20this%20text.*%0A%0A%3E%20Approximately%20how%20long%20do%20you%20think%20the%20post%20will%20be%3F%0A%0A*Replace%20this%20text.*%0A%0A%3E%20Approximately%20how%20long%20do%20you%20think%20it%20will%20take%20you%20to%20write%20this%20post%3F%0A%0A*Replace%20this%20text.*%0A%0A%3E%20Do%20you%20think%20your%20post%20could%20make%20use%20of%20any%20illustrations%20or%20photographs%3F%0A%0A*Replace%20this%20text.*%0A%0A%3E%20Do%20you%20have%20any%20links%20to%20previous%20writing%3F%0A%0A*Replace%20this%20text.*%0A%0A%3E%20Where%20else%20can%20we%20find%20you%20on%20the%20internet%3F%20%28e.g.%20your%20blog%2C%20Tumblr%2C%20Medium%2C%20Twitter%2C%20etc.%29%0A%0A*Replace%20this%20text.*%0A%0A%3E%20Anything%20else%20you%20want%20to%20tell%20us%3F%0A%0A*Replace%20this%20text.*%0A%60%60%60) and it will automatically create an issue with a markdown file of suggested questions to answer.

To create the `title=` and `body=` portions of the URL, paste the title text and body text (separately) here: http://meyerweb.com/eric/tools/dencoder/, press `encode` to get the URL encoded text and form the URL.

Might we do this for each of the pitch templates, meaning the issue can be auto created with the pertinent subheadings?

**Comment by ashleygwilliams** [comment]
WHOA that is brilliant, @Charlotteis. yes yes yes

**Comment by sup** [comment]
@Charlotteis that sounds amazing, i did not even know that existed.

**Comment by Charlotteis** [comment]
> @Charlotteis that sounds amazing, i did not even know that existed.

all thanks to my lovely Hoodie-colleague [at]gr2m

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -1,9 +1,110 @@
...
 
 _TODO: Add starting points_
 
+## Filing Issues
+
+Issues filed for this repository must meet the following requirements and structure.
+
+### Requirements
+
+The Inclusivity Working Group exists to increase inclusivity and diversity within the Node.js collaborator base. These discussions often touch on a wide range of topics which makes it easy for discussions to get off topic. The following requirements exist for filing issues to help make sure discussions stay on topic.
```
Comment (beaugunderson) [inline CONTRIBUTING.md]: thoughts on changing "collaborator base" to "community"?

Comment (sup) [inline CONTRIBUTING.md]: depends - did we clarify what our scope is yet? is it only the collaborators, or the entire community?

Comment (beaugunderson) [inline CONTRIBUTING.md]: unclear what "These discussions" are; maybe say "Discussions within the Working Group"?

(also, maybe change "get off topic" to "stray off-topic"?)

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -1,9 +1,110 @@
...
+
+- Issues should be filed with the intention of actionable steps being taken to further the mission of this working group
```
Comment (beaugunderson) [inline CONTRIBUTING.md]: clearer: "Issues should be filed with the intent to further the mission of this working group"

an aside: is working group always capitalized? should be consistent

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -1,9 +1,110 @@
...
+    - Issues in this repository are not the place for individual education about inclusivity/diversity, discussing whether or not this group should exist, etc.
```
Comment (beaugunderson) [inline CONTRIBUTING.md]: "whether or not" to "whether"

File: CONTRIBUTING.md [hunk 16]
```diff
@@ -1,9 +1,110 @@
...
+- Issues should be filed for topics that are within the scope of this Working Group
+    - For example, if you want to file an issue with project-wide scope, it should probably be filed under the [Technical Stearing Comittee repository](https://github.com/nodejs/tsc). Issues should only be filed that request changes at the TSC level here if it is an inclusivity/diversity related topic _and_ members of this working group are needed to spearhead the initiative.
+- Issues should not be filed for topics that have already been decided
```
Comment (beaugunderson) [inline CONTRIBUTING.md]: totally get why this is here; but I wonder if adding something like "without substantial new information or change in circumstances" might be prudent?

(I can see running into this if we want to change our minds about something, for example)

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -1,9 +1,110 @@
...
+    - Not everyone will agree with everything that has been decided by this group, this is expected.
+    - Issues that are filed in an attempt to reopen a discussion that has already been decided will be closed without discussion.
```
Comment (beaugunderson) [inline CONTRIBUTING.md]: maybe change "will be closed" to "will be closed immediately" because the next section talks about how discussions will remain open for a week which may lead to rule nit-picking

File: CONTRIBUTING.md [hunk 18]
```diff
@@ -1,9 +1,110 @@
...
+
+### Structure
+
+Each issue should have the following information.
+
+#### Purpose
+
+The purpose should be a brief description of what you would like to see addressed. The type of issue should be explicitly mentioned in the purpose:
+
+- _Question_ Some of the policies and recommendations of this group may not be clear, and a question asks for clarification. The resolution to a question should be an answer in the issue, not a change to policy or documentation.
```
Comment (beaugunderson) [inline CONTRIBUTING.md]: lots of overlap between "Question" and "Request for changes to a policy"; if someone is asking a question because the policy is unclear then the result might be us answering the question in the issue but also clarifying the wording in the policy itself?

File: CONTRIBUTING.md (hunk 12 above)
Comment (beaugunderson) [inline CONTRIBUTING.md]: :+1: on requesting new programs, maybe also a "Request for involvement"? (which might be help with moderation, help with inclusivity for a node event, etc.)

File: CONTRIBUTING.md [hunk 19]
```diff
@@ -1,9 +1,110 @@
...
+- _Request for changes to a policy_ A request for change is used to clarify an existing policy. Issues of this type are typically of the form "This document is unclear in the case of x. Can we modify it to be more explicit?" or "The document says x, but I think it would be more effective if we did y instead".
+- _Request for new policy_ This WG's equivalent of a feature request, a request to create new policies.
+
+In some cases, a question may lead to a request for changes to a policy or request for new policy. In this case, issue authors may either file a new issue or rewrite the original issue to match the format for the new issue, at their discretion.
```
Comment (beaugunderson) [inline CONTRIBUTING.md]: this kind of addresses my comment above re: overlap between questions and requests for changes; i think maybe it's more burden than necessary on the requestor because the formats/goals of the two are so similar?

File: CONTRIBUTING.md [hunk 20]
```diff
@@ -1,9 +1,110 @@
...
+
+#### Proposal
+
+This optional section only applies to policy issues.
+
+If you have an idea of what the policy should look like, include a summary of it here. This does not need to be a fully formed version of the policy, just an overview of what you think it should look like. If you have a more fully formed proposal, please submit it as a pull request.
+
+#### Focus
+
+This section defines what is "on topic" and what is "off topic." The discussions in this group touch on a broad range of topics, and it's very easy to get off topic no matter how well intentioned everyone is. By having an explicit definition of what is considered on topic and off topic, and what the issue author hopes to get out of the discussion, we should be able to focus better.
```
Comment (beaugunderson) [inline CONTRIBUTING.md]: "well intentioned" should be hyphenated

File: CONTRIBUTING.md (hunk 11 above)
Comment (beaugunderson) [inline CONTRIBUTING.md]: :+1: for using `>` to block quote

File: CONTRIBUTING.md [hunk 21]
```diff
@@ -1,9 +1,110 @@
...
+
+### Examples
+
+Here is an example for a question issue:
```
Comment (beaugunderson) [inline CONTRIBUTING.md]: change "for" to "of"; maybe italicize "question" or make it clear that it's one of the types listed above some other way?

File: CONTRIBUTING.md [hunk 22]
```diff
@@ -1,9 +1,110 @@
...
+
+This discussion is specifically about the classification of hot dogs in the realm of sandwiches. Discussions that don't consider both hot dogs and sandwiches should be considered off topic.
+
+Some off topic examples include, but are not limited to:
+- What other foods are classified as sandwiches?
+- Are hot dogs classified as snacks or meals?
+- Are hot dogs considered to be an American invention, or are they also considered of German origin?
+```
+
+Here is an example for a request for changes to a policy issue:
```
Comment (beaugunderson) [inline CONTRIBUTING.md]: same as above re: "for"/"of" and making it clear that it's a type example from above

File: CONTRIBUTING.md [hunk 23]
```diff
@@ -1,9 +1,110 @@
...
+
+Some off topic examples include, but are not limited to:
+- What other foods are classified as sandwiches?
+- Are hot dogs classified as snacks or meals?
+- Are hot dogs considered to be an American invention, or are they also considered of German origin?
+- The validity of other sections of the Code of Conduct
+- The usefulness or necessity of Codes of Conduct in general
+```
+
+Here is an example for a request for new policy issue, with a proposal:
```
Comment (beaugunderson) [inline CONTRIBUTING.md]: and again :)

File: CONTRIBUTING.md [hunk 24]
```diff
@@ -1,9 +1,110 @@
-Even though we have a charter, this WG is essentially still in its early
-stages.
-If you are interested in helping shape this WG, have a look at the issues and PRs.
+Even though we have a charter, this WG is essentially still in its early stages. If you are interested in helping shape this WG, please have a look at the issues and PRs.
```
Comment (beaugunderson) [inline CONTRIBUTING.md]: - spell out Working Group at least the first time it appears
- take out "Even though have a charter, "

**Comment by beaugunderson** [comment]
ok, done with my review! :sparkles:

File: CONTRIBUTING.md (hunk 13 above)
Comment (nebrius) [inline CONTRIBUTING.md]: The charter refers to the "Node.js project," so I think we should change "collaborator base" to that.

+1 to the other recommendations

File: CONTRIBUTING.md (hunk 8 above)
Comment (nebrius) [inline CONTRIBUTING.md]: This wording is old and being replaced in https://github.com/nodejs/inclusivity/pull/88. TBH I'm not _exactly_ sure what a starting point is either :)

File: CONTRIBUTING.md (hunk 24 above)
Comment (nebrius) [inline CONTRIBUTING.md]: I went ahead and reverted this change here since it's all being rewritten in This wording is old and being replaced in #88

File: CONTRIBUTING.md (hunk 14 above)
Comment (nebrius) [inline CONTRIBUTING.md]: I like that wording.

Huh, I was amazingly inconsistent in this section with capitalization. Our other docs use "Working Group," so I'll switch to that.

File: CONTRIBUTING.md (hunk 16 above)
Comment (nebrius) [inline CONTRIBUTING.md]: Makes sense. What do you think about adding it as a sub bullet point?

File: CONTRIBUTING.md (hunk 17 above)
Comment (nebrius) [inline CONTRIBUTING.md]: +1

File: CONTRIBUTING.md (hunk 10 above)
Comment (nebrius) [inline CONTRIBUTING.md]: will do

File: CONTRIBUTING.md (hunk 16 above)
Comment (beaugunderson) [inline CONTRIBUTING.md]: :+1:

File: CONTRIBUTING.md (hunk 19 above)
Comment (nebrius) [inline CONTRIBUTING.md]: I went back and forth on this myself. Ultimately I decided to keep them separate because the issue can always be edited to change it from a question to a request for change. I was worried that if we didn't have a place for people to ask questions, then it might discourage people from getting involved.

What do you think about adding this to the beginning of the question description: "A question about any of the policies, programs, processes, etc of this Working Group." to broaden what can be asked?

File: CONTRIBUTING.md (hunk 11 above)
Comment (nebrius) [inline CONTRIBUTING.md]: omg that link is amazing @ashleygwilliams!!!

Unfortunately, doing `> ## Title` renders as

> ## Title

so it can't be copy pasted. I can do 80col hard wraps though.

File: CONTRIBUTING.md (hunk 23 above)
Comment (nebrius) [inline CONTRIBUTING.md]: guess who was copy-pasting :D

**Comment by nebrius** [comment]
Thanks everyone for the reviews, I know this one is a little denser than most :) I think I addressed everyone's concerns, PTAL!

File: CONTRIBUTING.md [hunk 25]
```diff
@@ -4,6 +4,134 @@ If you are interested in helping shape this WG, have a look at the issues and PR
...
+
+Each issue should have the following information.
+
+#### Purpose
+
+The purpose should be a brief description of what you would like to see addressed. The type of issue should be explicitly mentioned in the purpose:
+
+- _Question_ A question about any of the policies, programs, processes, etc of this Working Group. Some of the policies and recommendations of this group may not be clear, and a question asks for clarification. The resolution to a question should be an answer in the issue, not a change to policy or documentation.
+- _Request for changes to a policy_ A request for change is used to clarify an existing policy. Issues of this type are typically of the form "This document is unclear in the case of x. Can we modify it to be more explicit?" or "The document says x, but I think it would be more effective if we did y instead".
+- _Request for new policy_ This Working Group's equivalent of a feature request, a request to create new policies.
```
Comment (Charlotteis) [inline CONTRIBUTING.md]: @ashleygwilliams mentioned something about 'Request for a new program', if we can get an idea of what that should look like, would it be a good thing to mention here as a bullet point too?

File: CONTRIBUTING.md [hunk 26]
```diff
@@ -4,6 +4,134 @@ If you are interested in helping shape this WG, have a look at the issues and PR
...
+
+In some cases, a question may lead to a request for changes to a policy or request for new policy. In this case, issue authors may either file a new issue or rewrite the original issue to match the format for the new issue, at their discretion.
+
+#### Proposal
+
+This optional section only applies to policy issues.
+
+If you have an idea of what the policy should look like, include a summary of it here. This does not need to be a fully formed version of the policy, just an overview of what you think it should look like. If you have a more fully formed proposal, please submit it as a pull request.
```
Comment (Charlotteis) [inline CONTRIBUTING.md]: 'include a summary of it here'

Where is here? :)

**Comment by Charlotteis** [comment]
Here is a magic URL I made to show how we could help people make 'Question' issues [click](https://github.com/nodejs/inclusivity/issues/new?title=Question%3A%20Are%20Hot%20Dogs%20considered%20sandwiches%3F&body=%23%23%20Purpose%0A%0AI%20looked%20through%20the%20inclusivity%20docs%2C%20specifically%20the%0A%5BCode%20of%20Conduct%5D(https%3A%2F%2Fgithub.com%2Fnodejs%2Finclusivity%2Fblob%2Fmaster%2FCODE_OF_CONDUCT.md)%0Aand%20it%27s%20unclear%20to%20me%20if%20hot%20dogs%20are%20considered%20sandwiches%20or%20not.%20So%20my%0Aquestion%20is%3A%20does%20the%20Code%20of%20Conduct%20consider%20hot%20dogs%20to%20be%20sandwiches%3F%20The%0Awording%20around%20this%20topic%20is%20unclear%20to%20me.%0A%0A%23%23%20Focus%0A%0AThis%20discussion%20is%20specifically%20about%20the%20classification%20of%20hot%20dogs%20in%20the%0Arealm%20of%20sandwiches.%20Discussions%20that%20don%27t%20consider%20both%20hot%20dogs%20and%0Asandwiches%20should%20be%20considered%20off%20topic.%0A%0ASome%20off%20topic%20examples%20include%2C%20but%20are%20not%20limited%20to%3A%0A-%20What%20other%20foods%20are%20classified%20as%20sandwiches%3F%0A-%20Are%20hot%20dogs%20classified%20as%20snacks%20or%20meals%3F%0A-%20Are%20hot%20dogs%20considered%20to%20be%20an%20American%20invention%2C%20or%20are%20they%20also%0A%20%20considered%20of%20German%20origin%3F)

**Comment by beaugunderson** [comment]
@Charlotteis those URLs are totally going to come in handy, thank you :)

**Comment by ashleygwilliams** [comment]
@nebrius this still needs to be moved out of `CONTRIBUTING.md` and into `docs/issues.md` or something like that

**Comment by nebrius** [comment]
> @nebrius this still needs to be moved out of CONTRIBUTING.md and into docs/issues.md or something like that

D'oh! Last run through I only read the inline comments, sorry! 😅

> Here is a magic URL I made to show how we could help people make 'Question' issues click

I love the magic link idea! Is it better to have the example pasted in (which users have a reference too but have to delete), or to have more of a template (with placeholders such as "_fill in purpose here_") and have the magic links right above the example in POLICY_ISSUES.md that they can refer back to?

**Comment by nebrius** [comment]
Regarding programs, I kinda feel like it might help to get some of the basics for programs down first before setting up issue guidelines (what do they look like, what is the process like for getting new programs going, etc), in the form of new policies, but what does everyone else think?

**Comment by nebrius** [comment]
ok, I tweaked the intro wording to hopefully make it more friendly and approachable, and I moved everything into docs/POLICY_ISSUES.md.

I still would like to get clarification on programs, so let's not consider this ready to merge yet.

Also, @beaugunderson I responded to some of your questions inline in the diff, and since pushed changes that marked those as "commented on an outdated diff". When you get a chance, can you let me know what you think?

**Comment by sup** [comment]
if we do get one of these magic URLs (which i'm absolutely for), then let's also put it somewhere in `CONTRIBUTING.md`. there's this little thing when creating a new issue from blank:

screen shot 2016-01-19 at 19 14 38

if, hypothetically, someone were to create an issue from scratch, they might be clicking the link to review the contributing guide, which is where they might also see the new magic URL.

*tl;dr: put the magic link in CONTRIBUTING.md, it helps guide issue creators the right way*

File: CONTRIBUTING.md (hunk 19 above)
Comment (beaugunderson) [inline CONTRIBUTING.md]: i like that :)

**Comment by beaugunderson** [comment]
@nebrius just responded to the one thing that i hadn't responded to that was still an open question, i think :)

**Comment by Charlotteis** [comment]
> I love the magic link idea! Is it better to have the example pasted in (which users have a reference too but have to delete), or to have more of a template (with placeholders such as "fill in purpose here") and have the magic links right above the example in POLICY_ISSUES.md that they can refer back to?

Probably better to have 'fill purpose here' unless we want people to delete several paragraphs about hot dogs every time they open an issue :)

**Comment by ashleygwilliams** [comment]
a++ i think this is good to merge. we can continue to iterate, but this is an great start and @Charlotteis already used it which is awesome. thoughts?

**Comment by nebrius** [comment]
Let me get the magic links included in the doc, rebase, and then I think we're good to go! I'm thinking we can wait on magic links in CONTRIBUTING.md for now cause of #88.

**Comment by sup** [comment]
fine by me, i'll open a separate PR when it's time

**Comment by sup** [comment]
LGTM! :sunglasses:

**Comment by ashleygwilliams** [comment]
wooo, merging. awesome job for spearheading this @nebrius -- and thanks to everyone for participating so thoughtfully!
